src/android/java/com/pspdfkit/cordova/PSPDFCordovaPlugin.java

```diff
@@ -64,7 +65,7 @@
         }
 
         if (TextUtils.isEmpty(licenseKey)) {
-            throw new PSPDFCordovaPluginException("PSPDFKit license key is missing! Please add a <meta-data android:name=\"PSPDFKIT_LICENSE_KEY\" android:value=\"...\"> to your AndroidManifest.xml.");
+            throw new PSPDFCordovaPluginException("PSPDFKit license key is missing! Please add a <meta-data android:name=\"pspdfkit_license_key\" android:value=\"...\"> to your AndroidManifest.xml.");
```

Comment:
The plugin class itself is called `PSPDFCordovaPlugin` since its base class is the `CordovaPlugin`. To avoid confusion, I'd either keep the prefix on the plugin (and on the exception) or replace it with `PSPDFKit`.